src/modules/audio/audio.c

```diff
@@ -210,6 +356,8 @@ bool lovrAudioInitDevice(AudioType type) {
   config.playback.channels = OUTPUT_CHANNELS;
   config.capture.channels = CAPTURE_CHANNELS;
   config.dataCallback = callbacks[type];
+  config.periodSizeInFrames = PERIOD_LENGTH;
+  config.periods = 3;
```

Comment:
In my tests last year on Quest, setting these gave me buffers which were always a multiple of PERIOD_LENGTH. Testing on Windows this does not happen, it ignores what I tell it and gives me buffers of length 441 always, which means we will always be a little delayed.

Reply:
Given that it is unreliable, should we still set it?  It looks like the current fixed buffer implementation is set up to work with any callback size.  If the only benefit to setting the period config is to sometimes get good callback sizes on certain backends, maybe we should instead just leave it unset and let miniaudio do its thing.  I trust miniaudio to pick better buffer sizes/timing intervals than lovr can (it's possible this is just a reflection of me not understanding this stuff well enough though).

Either way, explicitly setting the periods here overrides the `config.performanceProfile` setting below, because the performance profile is just a higher level way to configure it.  Seems like we should pick one or the other.

Reply:
If it is unreliable on PC but reliable on Quest, that still makes it useful on Quest. It also might be it can be made reliable on PC, or possibly reliability depends on some weird system quirk like which Windows sound backend is being used.